src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -435,10 +431,6 @@ def process_inputs(self, inputs: TextGenerationInput) -> List[numpy.ndarray]:
             inputs.sequences = repeat_inputs(
                 inputs.sequences, inputs.num_generated_predictions
             )
-            if self.engine:
-                self.engine.deterministic = False
-            if self.multitoken_engine:
-                self.multitoken_engine.deterministic = False
```

Comment:
we should replace this by updating the `TokenGenerator` args right?

Reply:
Where do we set deterministic to be False for this case then? @horheynm